### ISSUE TITLE
Better UI: Choose color, komi, handicap, start new game, etc.

### ISSUE THREAD

**Issue by maxpumperla** [maxpumperla/betago#6]

**Comment by maxpumperla** [comment]
@jokkebk We're making heavy use of your awesome jgoban lib here, but our demo is still underwhelming. Would you be interested in helping us, thereby testing your UI by seeing it in action?

**Comment by jokkebk** [comment]
Haha, thanks for the compliments, and wonderful to see you're using jGoBoard!

I'm unfortunately a bit pressed on time currently, so cannot offer much help with the AI project, but let me know if you'd need some features on jGoBoard side of things. My friend is currently doing a simple go server so we're looking at things like implementing scoring etc. there.

**Comment by ddugovic** [comment]
I'm curious (now that this engine has a GTP front end) whether it would make sense to somehow utilize [NNGS](http://github.com/matsu911/nngs) as a web server.

**Comment by maxpumperla** [comment]
Sure, definitely something we have to look into. I have no experience with NNGS, but would be very interested in this topic. @ddugovic have you done some work with NNGS? Maybe this calls for another issue, this was intended to be front-end only.

**Comment by maxpumperla** [comment]
@jokkebk It was worth a shot, wouldn't you say? :) I'll reach out to you if there's something that'd be cool to add. And w.r.t. the Go server, the "scoring" is what it's all about, so we could potentially work on that together.

**Comment by ddugovic** [comment]
@maxpumperla I managed to install [nngs](http://github.com/ddugovic/nngs) and connect!  I look forward to testing it, although I can't find the IGS/NNGS help pages.

**Comment by maxpumperla** [comment]
@ddugovic That's pretty cool! Can you somehow share your knowledge / document what and how you did that? Could be very valuable for a lot of people.

**Comment by ddugovic** [comment]
Yes, I'm working on it... I have updated the [README](http://github.com/ddugovic/nngs.git) file although I'm trying to obtain help files for the server so after I log in I can verify the installation...

**Comment by ddugovic** [comment]
It looks like my nngs installation is successful and I can connect with either telnet or clients like [Jago](http://senseis.xmp.net/?Jago).

**Comment by soma-six** [comment]
I potentially have a beautiful solution.... 
Demo: http://sabaki.yichuanshen.de/web/view/
Repo: https://github.com/yishn/sabaki-website
GTP: https://github.com/yishn/Sabaki/wiki/Engines

**Comment by rocketinventor** [comment]
@ddugovic NNGS hasn't been updated in a long time. If Betago's back end can handle these options, then they should be handled on the front end and it should be trivial to add them.

**Comment by rocketinventor** [comment]
Right now, there is no need for Komi, because there is no score-keeping going on (that I know of). Does anybody know of a library (python _or_ js) that can keep track of score?
